packages/react-component-library/src/components/NumberInput/constants.ts

```diff
@@ -3,4 +3,9 @@ const END_ADORNMENT_TYPE = {
   INCREASE: 'increase',
 } as const
 
-export { END_ADORNMENT_TYPE }
+const UNIT_POSITION = {
+  AFTER: 'after',
+  BEFORE: 'before',
+} as const
+
+export { END_ADORNMENT_TYPE, UNIT_POSITION }
```

Comment:
I don't think these are currently exported by the adjacent `index.ts` file. Is that an issue for consumers?